shoplist/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
```

Comment:
Stable API 31 is available. Consider testing the app on that api as well and change the target version